src/openrct2/scripting/ScEntity.hpp

```diff
@@ -56,8 +58,15 @@ namespace OpenRCT2::Scripting
                     case EntityType::Vehicle:
                         return "car";
                     case EntityType::Guest:
+                        if (targetApiVersion <= 33)
```

Comment:
Perhaps we should make a constant for 33, e.g. `API_VERSION_33_PEEP_DEPRECATION`.